docs/firestore/usage.mdx

```diff
@@ -780,3 +780,49 @@ FirebaseFirestore.instance.settings =
 
 // ...
 ```
+
+
+## Data Bundles
+
+If you have [setup a data bundle](https://firebase.google.com/docs/firestore/bundles#building_data_bundles_on_the_server) and are
+[currently serving it](https://firebase.google.com/docs/firestore/bundles#serving_data_bundles). You may load data bundles
+into your app.
+
+```dart
+// use a package like 'http' to retrieve bundle.
+import 'package:http/http.dart' as http;
+final response = await http.get(url);
+
+// convert the 'bundle.txt' string in the response to an Uint8List instance.
+Uint8List buffer = Uint8List.fromList(response.body.codeUnits);
+
+//Load bundle into cache
+LoadBundleTask task = firestore.loadBundle(buffer);
+
+//use .stream() API to expose a stream which listens for LoadBundleTaskSnapshot events
+task.stream().listen((taskStateProgress) {
```

Comment:
The doc probably should mention

```dart
await task.stream().last;
```

to wait for the bundle to be fully loaded



In the end, `listen` is only useful if you want to show a loading bar that tracks the progress, which is quite niche.